src/components/ResultTable.tsx

```diff
@@ -61,59 +63,74 @@ const ResultTable: React.FC<Props> = ({ simuleringsresultat }) => {
-          </Table.Row>
-          {pensjonsalder.map((alder, index) => (
-            <Table.Row key={index}>
-              <Table.DataCell>{alder}</Table.DataCell>
-              <Table.DataCell>{alderspensjonData[index]}</Table.DataCell>
+      <div className={stepStyles.tabellScrollbarhet}>
+        <Table data-testid="result-table">
+          <Table.Header>
+            <Table.Row>
+              <Table.HeaderCell scope="col">Alder</Table.HeaderCell>
+              <Table.HeaderCell scope="col">
+                Alderspensjon (Nav)
+              </Table.HeaderCell>
+              <Table.HeaderCell scope="col">AFP privat</Table.HeaderCell>
+              <Table.HeaderCell scope="col">
+                Pensjonsgivende inntekt
+              </Table.HeaderCell>
+              <Table.HeaderCell scope="col">Sum</Table.HeaderCell>
+            </Table.Row>
+          </Table.Header>
+          <Table.Body>
+            <Table.Row>
+              <Table.HeaderCell scope="row">
+                {pensjonsalder.length > 0 ? `${pensjonsalder[0] - 1} år` : 0}
+              </Table.HeaderCell>
+              <Table.DataCell>0</Table.DataCell>
+              <Table.DataCell>0</Table.DataCell>
               <Table.DataCell>
-                {(afpPrivatData ? afpPrivatData[index] : 0) || 0}
+                {state.aarligInntektFoerUttakBeloep}
               </Table.DataCell>
               <Table.DataCell>
-                {inntektVsaGradertUttakInterval.includes(alder)
-                  ? aarligbelopVsaGradertuttak
-                  : inntektVsaHelPensjonInterval.includes(alder)
-                    ? aarligbelopVsaHeltuttak
-                    : 0}
-              </Table.DataCell>
-              <Table.DataCell>
-                {alderspensjonData[index] +
-                  ((afpPrivatData ? afpPrivatData[index] : 0) || 0) +
-                  (inntektVsaGradertUttakInterval.includes(alder)
-                    ? aarligbelopVsaGradertuttak
-                    : inntektVsaHelPensjonInterval.includes(alder)
-                      ? aarligbelopVsaHeltuttak
-                      : 0)}
+                {state.aarligInntektFoerUttakBeloep}
               </Table.DataCell>
             </Table.Row>
-          ))}
-        </Table.Body>
-      </Table>
+            {pensjonsalder.map((alder, index) => (
+              <Table.Row key={index}>
+                <Table.HeaderCell scope="row">
+                  {pensjonsalder.length - 1 === index
+                    ? `${alder}+ år (livsvarig)`
+                    : `${alder} år`}
+                </Table.HeaderCell>
+                <Table.DataCell>
+                  {formatInntekt(alderspensjonData[index])}
+                </Table.DataCell>
+                <Table.DataCell>
+                  {afpPrivatData && afpPrivatData.length > 0
+                    ? formatInntekt(afpPrivatData[index]) ||
+                      formatInntekt(afpPrivatData[afpPrivatData.length - 1])
+                    : 0}
```

Comment:
Du har glemt å oppdatere siste kolonne tilsvarende. For å unngå duplisert logikk og redusere sjansen for feil, bør verdien legges i en const som kan gjenbrukes i siste kolonne.

Reply:
Dette burde vel forresten også dekkes av en test?